etk-asm/src/ops/expression.rs

```diff
@@ -0,0 +1,420 @@
+use super::macros::{ExpressionMacroInvocation, MacroDefinition};
+use num_bigint::BigInt;
+use snafu::OptionExt;
+use snafu::{Backtrace, Snafu};
+use std::collections::HashMap;
+use std::fmt::{self, Debug};
+
+/// An error that arises when an expression cannot be evaluated.
+#[derive(Snafu, Debug)]
+#[snafu(visibility = "pub")]
```

Comment:
```suggestion
#[snafu(visibility = "pub(crate)")]
```

Unless other crates are supposed to be creating `Error` instances?

Reply:
fixed